blocklylib-core/src/main/java/com/google/blockly/android/control/BlocklyController.java

```diff
@@ -65,20 +66,13 @@
 /**
  * Controller to coordinate the state among all the major Blockly components: Workspace, Toolbox,
  * Trash, models, and views.
```

Comment:
You should add a comment that All methods must be run on the main thread unless specified otherwise. After this change, most of the public methods will crash if called from another thread.